app/assets/json/translations/react/en.json

```diff
@@ -875,6 +875,8 @@
   "listings.occupancyDescriptionSomeSro": "Occupancy for this building varies by unit type. SROs are limited to 1 person per unit, regardless of age. For all other unit types, occupancy limits do not count children under 6.",
   "listings.openWaitlist": "Open Waitlist",
   "listings.openWaitlistSlots": "Open Waitlist Slots",
+  "listings.minPeople": "at least %{num} people",
```

Comment:
are these translations added to the google doc?

Reply:
yes! under the occupancy table section